### DISCUSSION TITLE
How to download external resources within the URL that is gonna be downloaded?

### DISCUSSION THREAD

**Discussion by the-prodigy** [website-scraper/node-website-scraper#448] [General]
Hello!

How to download external resources within the URL that is gonna be downloaded?

Like how do I download the external resources from URLs I can choose and select and then bind them to the website that I want to download?
@s0ph1e 
Best Regards!

**Comment by s0ph1e** [comment]
Hi @the-prodigy 👋 

I'm not sure I understood your question. Could you please clarify and maybe provide an example of what you want to achieve? 

All the external resources should be downloaded automatically by default if original resource has references to them.

**Reply by the-prodigy** [reply]
No my website resources dont get downloaded by default, I had to deal with urlFilter to download these files from a Domain.

Google Fonts etc within the destination URL are still not downloaded and always loaded from the original external resources.

Do u have discord? Better place

**Reply by s0ph1e** [reply]
@the-prodigy no, don't have a discord. 
If you can share your options - I can try to help. Or you can [enable logs](https://github.com/website-scraper/node-website-scraper#log-and-debug) to check if your resources are filtered out by some rules.
